scripts/add_attribute.py

```diff
@@ -83,104 +82,132 @@
+            inInit = True
+        if inInit:
+            if not line or line.endswith(" = github.GithubObject.NotSet") or line.endswith(" = NotSet"):
+                if line:
+                    attrName = line[14:-29]
+                if not line or attrName > attributeName:
+                    newLines.append(f"        self._{attributeName}: Attribute[{attributeClassType}] = NotSet")
+                    added = True
+        newLines.append(line)
+
+    while i < len(lines):
         line = lines[i].rstrip()
-    except IndexError:
-        line = ""
-    i += 1
-    if line == "    def _useAttributes(self, attributes:Dict[str,Any]) -> None:":
-        inUse = True
-    if inUse:
-        if not line or line.endswith(" in attributes:  # pragma no branch"):
-            if line:
-                attrName = line[12:-36]
-            if not line or attrName > attributeName:
-                newLines.append('        if "' + attributeName + '" in attributes:  # pragma no branch')
-                if attributeAssertType:
+        i += 1
+        newLines.append(line)
+
+    return newLines
+
+
+def add_useAttributes(lines: list[str]) -> list[str]:
+    i = 0
+    newLines = []
+    added = False
+    inUse = False
+    while not added:
+        try:
+            line = lines[i].rstrip()
+        except IndexError:
+            line = ""
+        i += 1
+        if line.strip().startswith("def _useAttributes(self, attributes:"):
+            inUse = True
+        if inUse:
+            if not line or line.endswith(" in attributes:  # pragma no branch"):
+                if line:
+                    attrName = line[12:-36]
+                if not line or attrName > attributeName:
+                    newLines.append('        if "' + attributeName + '" in attributes:  # pragma no branch')
+                    if attributeAssertType:
+                        newLines.append(
+                            '            assert attributes["'
+                            + attributeName
+                            + '"] is None or isinstance(attributes["'
+                            + attributeName
+                            + '"], '
+                            + attributeAssertType
+                            + '), attributes["'
+                            + attributeName
+                            + '"]'
+                        )
                     newLines.append(
-                        '            assert attributes["'
-                        + attributeName
-                        + '"] is None or isinstance(attributes["'
-                        + attributeName
-                        + '"], '
-                        + attributeAssertType
-                        + '), attributes["'
-                        + attributeName
-                        + '"]'
+                        f"            self._{attributeName}: Attribute[{attributeClassType}] = {attributeValue}"
```

Comment:
We don't have the typing in `_useAttributes`:
```suggestion
                        f"            self._{attributeName} = {attributeValue}"
```